### ISSUE TITLE
Build hangs after docker login

### ISSUE THREAD

**Issue by caarlos0** [Shippable/support#1409]
I got:

```
WARNING: login credentials saved in /root/.dockercfg.
Login Succeeded
```

Example build: https://app.shippable.com/builds/5547bafb42da970c000f1042

The build still in 'processing' mode _ad aeternum_.

Screenshot:

![screenshot from 2015-05-04 16 21 31](https://cloud.githubusercontent.com/assets/245435/7460480/b5c35584-f279-11e4-9a09-e921ba7d6364.png)

BTW: I cancelled all the builds in this scenario to see if it somehow improves something, but it [hasn't fixed it](https://app.shippable.com/builds/5547ca4542da970c000f11b0).

**Comment by fdaciuk** [comment]
+1

**Comment by ladraum** [comment]
+1

**Comment by manishas** [comment]
we'll take a look.. i am assuming this is all for ContaAzul.

**Comment by caarlos0** [comment]
@manishas yes it is.

**Comment by caarlos0** [comment]
@manishas any news on this?

can it be somehow related to free space? I cleaned all our custom servers up...

**Comment by andersonborges** [comment]
+1

**Comment by Cunhazera** [comment]
+1

**Comment by gustavjr** [comment]
+1

**Comment by decioluckow** [comment]
+1

**Comment by tadeubonatti** [comment]
+1

**Comment by celinoaraujo** [comment]
+1

**Comment by rafaelcamargo** [comment]
+1

**Comment by matheusrossato** [comment]
+1

**Comment by andersonresendes** [comment]
+1

**Comment by AlissonMedeiros** [comment]
+1

**Comment by matheuspoleza** [comment]
+1

**Comment by robsonmafra** [comment]
+1

**Comment by manishas** [comment]
we're pushing a fix for this today. i'll update as soon as it's deployed.

**Comment by caarlos0** [comment]
@manishas many thanks :beers:

**Comment by caarlos0** [comment]
@manishas any updates? We still with a lot of issues here :/

**Comment by Manc** [comment]
I can't build anything with a dedicated host (it does work with Shippable's multi-tenant environment). I tried both a new AWS EC2 node and on-premise with a freshly installed Ubuntu 14.04 (init script ran without a problem).
It looks exactly the same as described and shown in this ticket. After a while I simply get a timeout.
I've tried both a Scala project and a tiny PHP project (build_image: shippable/minv2).
